pkg/controller/installation/products/config/rhsso.go

```diff
@@ -0,0 +1,27 @@
+package config
+
+import "github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+
+type RHSSO struct {
+	config ProductConfig
+}
+
+func newRHSSO(config ProductConfig) *RHSSO {
+	return &RHSSO{config: config}
+}
+
+func (a *RHSSO) GetNamespace() string {
```

Comment:
can we change the `a *RHSSO` to `r *RHSSO` as this could get confusing if this class grows in complexity.

Reply:
this applies to all the methods on this struct.

Reply:
:+1: thumb

Reply:
https://github.com/integr8ly/integreatly-operator/pull/18/commits/cd425b8b5a95b7b51ddfac0de8e1d92c3110ae68